mungegithub/mungers/approvers/approvers_test.go

```diff
@@ -656,3 +663,82 @@ You can cancel your approval by writing ` + "`/approve cancel`" + ` in a comment
 		t.Errorf("GetMessage() = %+v, want = %+v", *got, want)
 	}
 }
+
+func TestGetMessageApprovedIssueAssociated(t *testing.T) {
+	ap := NewApprovers(
+		Owners{
+			filenames: []string{"a/a.go", "b/b.go"},
+			repo: createFakeRepo(map[string]sets.String{
+				"a": sets.NewString("Alice"),
+				"b": sets.NewString("Bill"),
+			}),
+		},
+	)
+	ap.AssociatedIssue = 12345
+	ap.AddAuthorSelfApprover("John", "REFERENCE")
+	ap.AddApprover("Bill", "REFERENCE", false)
+	ap.AddApprover("Alice", "REFERENCE", false)
+
+	want := `[APPROVALNOTIFIER] This PR is **APPROVED**
```

Comment:
Ya, if the author is an OWNER, we can allow typing `/approve no-issue`.
Otherwise, we can have another approver to do so.